src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -62,7 +62,8 @@
 #define MLOG_PEER_STATE(x) \
   MCINFO(MONERO_DEFAULT_LOG_CATEGORY, context << "[" << epee::string_tools::to_string_hex(context.m_pruning_seed) << "] state: " << x << " in state " << cryptonote::get_protocol_state_string(context.m_state))
 
-#define BLOCK_QUEUE_NSPANS_THRESHOLD 10 // chunks of N blocks
+#define BLOCK_QUEUE_NSPANS_THRESHOLD 200 // chunks of N blocks
+#define BLOCK_QUEUE_NSPANS_MINIMUM 10 // minimum number of spans                                        // 
```

Comment:
This dangling `//` is the only (non-)issue I see in this PR.,

Reply:
Thanks for mentioning it. Fixed.